src/docs/product/alerts/create-alerts/metric-alert-config.mdx

```diff
@@ -8,34 +8,25 @@ description: "Learn more about the options for configuring a metric alert."
 
 Sentry provides several configuration options to create a metric alert based on your organization's needs.
 
-## Filters
-
-The following set of filters translates into a [Discover query](/product/discover-queries/) that is displayed in the chart at the top of the alert configuration page.
-
-### Environment
-
-Specify which <PlatformLink to="/configuration/environments/">environment(s)</PlatformLink> will use this particular alert rule. This control filters on the `environment` tag in your events. This filter is helpful because the urgency and workflows you apply to production alerts might differ from those you apply to alerts originating from your QA environment, for example.
-
-The “Env:” dropdown list here has the same environments that are available for the selected project in the common “Environment” filter dropdown (this does not include hidden environments). Selecting "All" is equivalent to having no environment filter.
-
-### Event Type
-
-For some metric alerts, you can set the event type that you want to be alerted about in the “Events” dropdown:
+## Metric (Type + Function + Time Interval)
 
-- `event.type:error` OR `event.type:default`
-- `event.type:default`
-- `event.type:error`
-- `event.type:transaction`
+To create a metric alert you first need to choose a metric type. For some alert types the function is built into the alert and for others you can choose functions and parameters to apply to it. For example, if you select “Users Experiencing Errors”, that translates to the function, `count_unique(user.id)`. Since editing this function would change the nature of the alert, it is not editable and thus hidden. On the other hand, if you select “Largest Contentful Paint” the measurement used is `measurement.lcp`, you also need to choose a function, e.g. `p75()`, for a combined metric function of `p75(measurement.lcp)`.
 
-### Tags & Attributes
+### Metrics Types for Alerting
 
-Add filters in the field provided to narrow down what you'll be alerted about, such as URL, tags, or other event properties.
-
-## Metric (Function + Time Interval)
-
-Depending on the type of alert you’ve selected, you may be able to choose functions and parameters to apply to it. In other cases, the function is built into the alert and the settings aren't displayed. For example, if you select “Number of Users Affected”, that translates to the function, `count_unique(user.id)`. Since editing this function would change the nature of the alert, it is not editable and thus hidden.
+- `Number of Errors`
+- `Users Experiencing Errors`
+- `Crash Free Session Rate`
+- `Crash Free User Rate`
+- `Throughput`
+- `Transaction Duration`
+- `Apdex`
+- `Failure Rate`
+- `Largest Contentful Paint`
+- `First Input Delay`
+- `Cumulative Layout Shift`
```

Comment:
We don't need to use code styling here since these are just selections in a dropdown. We can leave the functions in code styling though since they're also queries.

```suggestion
- Number of Errors
- Users Experiencing Errors
- Crash Free Session Rate
- Crash Free User Rate
- Throughput
- Transaction Duration
- Apdex
- Failure Rate
- Largest Contentful Paint
- First Input Delay
- Cumulative Layout Shift
```